packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -599,8 +599,9 @@
   "Cloud_connect": "Rocket.Chat Cloud Connect",
   "Cloud_what_is_it": "What is this?",
   "Cloud_what_is_it_description": "Rocket.Chat Cloud Connect allows you to connect your self-hosted Rocket.Chat Workspace to our Cloud. Doing so enables you to manage your licenses, Billing and Support in Rocket.Chat Cloud.",
-  "Cloud_workspace_connected_plus_account": "Your workspace is now connected to the Rocket.Chat Cloud and an account is associated.",
-  "Cloud_workspace_connected_without_account": "Your workspace is now connected to the Rocket.Chat Cloud. If you would like, you can login to the Rocket.Chat Cloud and associate your workspace with your Cloud account.",
+  "Cloud_workspace_connected": "Your workspace has been successfully connected to Rocket.Chat Cloud.  You can access the cloud to manage account information",
+  "Cloud_workspace_support": "If you have any trouble with a cloud service please try to sync first.  If the issue persists please you can open a support ticket in the Cloud Console.",
+  "Cloud_workspace_disconnect": "If you no longer wish to receive cloud services you can disconnect your workspace from the Rocket.Chat Cloud.",
```

Comment:
Should this say `utilize` instead of `receive`?

Reply:
```suggestion
  "Cloud_workspace_disconnect": "If you no longer wish to utilize cloud services you can disconnect your workspace from the Rocket.Chat Cloud.",
```